QueAns/views.py

```diff
@@ -0,0 +1,91 @@
+from django.http import HttpResponse, HttpResponseBadRequest, HttpResponseForbidden
+from .models import Voting,Satisfaction,Question,Meetings
+from django.utils import timezone
+
+
+def upvote(request,user_id,question_id):
+    if request.user.is_authenticated():
+        if request.method == "POST":
+            meeting = Question.objects.filter(id=question_id).values_list('meeting_id')
+            meets_end_time = Meetings.objects.filter(id=meeting).values_list('end_time')
```

Comment:
This is still not what I wanted, this works but I was trying to make you learn something.
Ref :- https://docs.djangoproject.com/en/2.0/ref/models/querysets/#django.db.models.query.QuerySet.values

In the above link there is this text :- 

> The people who made Django prefer to put all the SQL-affecting methods first, followed (optionally) by any output-affecting methods (such as values()), but it doesn’t really matter. This is your chance to really flaunt your individualism. You can also refer to fields on related models with reverse relations through OneToOneField, ForeignKey and ManyToManyField attributes:
>>> Blog.objects.values('name', 'entry__headline')
<QuerySet [{'name': 'My blog', 'entry__headline': 'An entry'},
     {'name': 'My blog', 'entry__headline': 'Another entry'}, ...]>

Reply:
Also, please try to print SQL wrt to this queryset to see what is exact SQL django is trying to execute on DB and you'll learn JOINs :)